src/Amazon.php

```diff
@@ -74,11 +82,17 @@ class Amazon
      * @param  string $countryCode Country code for Amazon service; may be US, UK, DE, JP, FR, CA
      * @param  string $secretKey   API Secret Key
      * @param  string $version     API Version to use
+     * @param  bool   $useSsl      Use HTTPS instead of HTTP?
      * @throws Exception\InvalidArgumentException
      * @return Amazon
      */
-    public function __construct($appId, $countryCode = 'US', $secretKey = null, $version = null)
-    {
+    public function __construct(
+        $appId,
+        $countryCode = 'US',
+        $secretKey = null,
+        $version = null,
+        $useSsl = false
```

Comment:
I don't have strong preference either.  It is just that "SSL" irks me for unrelated reason - we don't really mean SSL transport, which is vulnerable and insecure, when we use that term.